polls/resource.py

```diff
@@ -87,26 +87,24 @@ def get(self, request, *args, **kwargs):
         return response
 
     def determine_content_type(self, request):
-        acceptable = (
-            AcceptParameters(ContentType('application/json')),
-            AcceptParameters(ContentType('application/hal+json')),
-            AcceptParameters(ContentType('application/vnd.hal+json')),
-            AcceptParameters(ContentType('application/vnd.siren+json')),
-        )
+        content_types = [
+            'application/vnd.siren+json',
+            'application/vnd.hal+json',
+            'application/hal+json',
+            'application/json',
```

Comment:
Just a note, mimeparse excepts the array to be ordered priortiy, last entries take priority. This is opposite for negotiatior and thus order is switched.